recipes/pypdf2_fields/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "PyPDF2_Fields" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyPDF2_Fields-{{ version }}.tar.gz
+  sha256: 017f0efcf9625b1e47d000e1c27442358169e98362a55d090280d201f0aaedc4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pypdf2 ==1.26.0
```

Comment:
Usually I'd prefer to put `>=1.26.0`, but as that package hasn't been updated since 2016, I guess it's fine like that.

Reply:
I prefer using ==.